packages/tools-iottwinmaker/README.md

```diff
@@ -90,6 +117,18 @@ For example:
 ```
 tmdt nuke --region us-east-1 --workspace-id SyncB
 ```
+
+Another example:
+
+The following will delete all entities, component types, scenes, workspace s3 bucket, and TwinMaker workspace itself.
+
+**Warning:** This command is destructive. Please take caution before running this command.
+```
+tmdt nuke --region us-east-1 --workspace-id SyncB --delete-workspace --delete-s3-bucket
+```
+**Additional notes:** 
+1. *--delete-s3-bucket* implies *--delete-workspace*; you will not be able to delete a workspace s3 bucket without deleting the workspace itself (however you may delete the workspace without deleting the s3 bucket)
```

Comment:
That actually is the case. If a user attempts to set delete-s3-bucket without workspace, the command will not run bc I set "implies()" in yargs command builder, it will prompt user "missing dependency delete-s3-bucket -> delete-workspace"

Reply:
Awesome, well designed